app/page.tsx

```diff
@@ -1,13 +1,21 @@
-import DynamicBadge from "@/components/Scenes/DynamicBadge";
+import DynamicBadge from "@/components/DynamicBadge";
 import { isMobileDevice } from "@/utils/isMobileDevice";
-import StaticBadge from "@/components/Scenes/StaticBadge";
+import StaticBadge from "@/components/StaticBadge";
 
 export const revalidate = 1200;
 
 export default function Home() {
   return (
-    <main className="flex h-screen w-full justify-center">
-      {isMobileDevice() ? <StaticBadge /> : <DynamicBadge />}
-    </main>
+    <>
+      {isMobileDevice() ? (
```

Comment:
Need to check if this is reliable. It doesn't seem to work well on dev tools in chrome when passing media queries.

Reply:
Will check this later. I'm lazy lol